src/raftstore/store/snap.rs

```diff
@@ -371,3 +383,56 @@ pub type SnapManager = Arc<RwLock<SnapManagerCore>>;
 pub fn new_snap_mgr<T: Into<String>>(path: T, ch: Option<SendCh<Msg>>) -> SnapManager {
     Arc::new(RwLock::new(SnapManagerCore::new(path, ch)))
 }
+
+#[cfg(test)]
+mod test {
+    use std::path::Path;
+    use std::fs::File;
+
+    use tempdir::TempDir;
+
+    use util::HandyRwLock;
+    use super::*;
+
+    #[test]
+    fn test_snap_mgr() {
+        let path = TempDir::new("test-snap-mgr").unwrap();
+
+        // mgr should create directory when not exist.
```

Comment:
Revise this comment to 
"`mgr` should create the specified directory when it does not exist."